uptane-standard.md

```diff
@@ -867,11 +867,19 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 11. For each image listed in the Targets metadata file from the Director repository, locate a Targets metadata file that contains an image with exactly the same file name. For each delegated Targets metadata file that is found to contain metadata for the image currently being processed, perform all of the checks in step 10. Use the following process to locate image metadata.  NOTE that if at any point below, a role you attempt to check for image metadata cannot be found and verified, you should abort the search and indicate that image metadata cannot be found because of a missing or invalid role.
     1. If the top-level Targets metadata file contains signed metadata about the image, return the metadata to be checked and skip to step 11.3.
     2. Recursively search the list of delegations, in order of appearance:
-        0. If the delegation does not address the image name being sought -- if none of the delegation's image paths match this image name -- skip this delegation.
-        1. If it is a multi-role delegation {{TAP-3}}, recursively visit each role, and check that each has signed exactly the same non-custom metadata (i.e., length and hashes) about the image. If it is all the same, return the metadata to be checked and skip to step 11.3.
-        2. If it is a terminating delegation and it contains signed metadata about the image, return the metadata to be checked and skip to step 11.3. If metadata about an image is not found in a terminating delegation, return an error code indicating that the image is missing.
-        3. Otherwise, continue processing the next delegation, if any. As soon as a delegation is found that contains signed metadata about the image, return the metadata to be checked and skip to step 11.3.
-        4. If no signed metadata about the image can be found anywhere in the delegation graph, return an error code indicating that the image is missing.
+        1. If the delegation does not address the image name being sought -- if none of the delegation's image paths match this image name -- skip this delegation.
+        2. If it is a multi-role delegation {{TAP-3}}, then:
+            1. First, recurse (11.2) for each role listed as if there is a single-role delegation to each.
+            2. Inspect the return values from each:
+                1. If at least a number of roles equal to the minimum number of roles in agreement for this multi-role delegation return the same non-custom metadata (i.e. length and hashes) for the image, then return that metadata for the image.  If the values returned are such that there are multiple sets of non-custom metadata meeting the threshold (e.g. if the minimum number of roles in agreement is two and two sets of at least two roles agree on different metadata), then choose the values returned by the set of roles that includes the earliest role in the multi-role delegation.
```

Comment:
The last sentence could be simplified, it's a bit hard to read

Otherwise LGTM abeit a bit complicated

Reply:
Does this read better?

*If at least a number of roles equal to the minimum number of roles in agreement for this multi-role delegation return the same non-custom metadata (i.e. length and hashes) for the image, then return that metadata for the image.  A conflict exists if there are multiple different sets of non-custom metadata with enough roles in agreement about them to meet the role threshold (for example, if the minimum number of roles in agreement is two and two sets of at least two roles agree on different metadata).  To resolve such a conflict, choose the metadata returned by the set of roles that includes the earliest role in the multi-role delegation.*

Reply:
Sounds good, thanks for making TAP 3 much clearer!